### PR TITLE
Implementing Edit Sayings UI

### PR DESCRIPTION
**Pull request by jsmorgan42**
Changes includes:
- Added settings option that links to the Edit Sayings page
- Created basic UI for Edit Sayings page
- Includes adding new collection view controller that contains a dynamic infinite scrolling flow layout
- Adding pagination view that contains buttons to paginate to previous & next page

Screen Shot 2020-03-18 at 10 45 14 AM

### CODE REVIEW

File: Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import UIKit
+import Combine
+
+class EditSayingsCollectionViewCell: VocableCollectionViewCell {
+    
+    @IBOutlet var textLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift]:
```suggestion
```

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//  Created by Thomas Shealy on 3/11/20.
+//  Copyright © 2020 WillowTree. All rights reserved.
+//
+
+import UIKit
+
+class EditSayingsViewController: UIViewController {
+
+    @IBOutlet var addButton: GazeableButton!
+    @IBOutlet var backButton: GazeableButton!
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]:
```suggestion
    @IBOutlet private var addButton: GazeableButton!
    @IBOutlet private var backButton: GazeableButton!
```

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]:
```suggestion
```

File: Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+//  Copyright © 2020 WillowTree. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Combine
+
+class EditSayingsCollectionViewCell: VocableCollectionViewCell {
+    
+    @IBOutlet var textLabel: UILabel!
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift]:
```suggestion
    @IBOutlet private var textLabel: UILabel!
```

**Review by Clstroud — APPROVED** [review]
Just a few minor suggestions -- some of them may not be warranted if they're just stubs that will be used in the next few PRs for this feature

File: Vocable/Common/CarouselGridCollectionViewController.swift [hunk 5]
```diff
@@ -0,0 +1,301 @@
+//
+//  ViewController.swift
+//  YouScrollMeRightRound
```
Comment (stevefosterwta) [inline Vocable/Common/CarouselGridCollectionViewController.swift:3]: Vocable AAC

File: Vocable/Common/Views/PaginationView.swift [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    override var intrinsicContentSize: CGSize {
+        if [traitCollection.horizontalSizeClass, traitCollection.verticalSizeClass].contains(.compact) {
+            return CGSize(width: 56, height: 48)
+        }
+        return CGSize(width: 104, height: 94)
+    }
+}
+
+@IBDesignable
+class PaginationView: UIView {
```
Comment (stevefosterwta) [inline Vocable/Common/Views/PaginationView.swift]: Should this be `final class`?

File: Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+//
+//  Created by Jesse Morgan on 3/11/20.
+//  Copyright © 2020 WillowTree. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Combine
+
+class EditSayingsCollectionViewCell: VocableCollectionViewCell {
```
Comment (stevefosterwta) [inline Vocable/Features/Settings/EditSayings/EditSayingsCollectionViewCell.swift]: `final class`

File: Vocable/Common/Views/PaginationView.swift [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+            textLabel.font = .systemFont(ofSize: 22, weight: .bold)
+            return
+        }
+        stackView.spacing = 42
+        textLabel.font = .systemFont(ofSize: 28, weight: .bold)
+    }
+
+    override func prepareForInterfaceBuilder() {
+        super.prepareForInterfaceBuilder()
+        textLabel.text = "Page 1 of 3"
```
Comment (stevefosterwta) [inline Vocable/Common/Views/PaginationView.swift:96]: Do we need to localize this?

**Pull request merged**